rmgpy/molecule/resonance.py

```diff
@@ -461,9 +461,18 @@ def generateAromaticResonanceStructures(mol, features=None):
     # Then determine which ones are aromatic
     aromaticBonds = molecule.getAromaticRings(rings)[1]
 
+    # Check if the bonds in at least one ring are alternating single/double
+    # If not, then we need to shift electrons around to obtain the true aromatic form
+    valid = False
+    for bonds in aromaticBonds:
+        orders = [bond.order for bond in bonds]
```

Comment:
I am slightly confused how this works. I thought `bond.order` was a float, so a list of floats should be returned (and `getOrderStr()` should be used to get the string form. Maybe I am missing something.

Reply:
That's true... I have no idea what I was doing, or why this appears to work.

Reply:
Thanks for pointing this out. Turns out that this wouldn't have worked anyway, since the bonds are not guaranteed to be in the correct order. I couldn't figure out a robust solution, so I decided to go with the assumption that it is always invalid instead.